tests/End2EndTestCase.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Tests\Delivery;
+
+use Contentful\Delivery\Client;
+use Contentful\Log\ArrayLogger;
+
+class End2EndTestCase extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var string
+     */
+    protected static $cacheDir = '';
+
+    public static function setUpBeforeClass()
+    {
+        self::$cacheDir = __DIR__ . '/../build/cache';
+    }
+
+    /**
+     * @param string $key
+     *
+     * @return Client
+     */
+    protected function getClient($key)
+    {
+        $options = getenv('CONTENTFUL_CDA_SDK_TESTING_URL')
+            ? ['uriOverride' => getenv('CONTENTFUL_CDA_SDK_TESTING_URL')]
+            : [];
+
+        switch ($key) {
```

Comment:
Well, same as before, you can look at the `ErrorTest.php` and see the space being used, it doesn't really change that 🙂